packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -594,6 +594,44 @@ const conf = module.exports = convict({
     env: 'STATIC_RESOURCE_URL',
     format: 'url'
   },
+  subscriptions: {
+    allowedLanguages: {
+      default: ['en'],
+      doc: 'Which languages are allowed to access subscription features?',
+      env: 'SUBSCRIPTIONS_ALLOWED_LANGUAGES',
+      format: Array,
+    },
+    enabled: {
+      default: false,
+      doc: 'Indicates whether subscriptions APIs are enabled',
+      env: 'SUBSCRIPTIONS_ENABLED',
+      format: Boolean,
+    },
+    managementClientId: {
+      default: '59cceb6f8c32317c',
+      doc: 'OAuth client ID for subscriptions management pages',
+      env: 'SUBSCRIPTIONS_MANAGEMENT_CLIENT_ID',
+      format: String,
+    },
+    managementScopes: {
+      default: 'profile https://identity.mozilla.com/account/subscriptions',
+      doc: 'The publicly visible URL of the subscription page/service',
+      env: 'SUBSCRIPTIONS_MANAGEMENT_SCOPES',
+      format: String,
+    },
+    managementTokenTTL: {
+      default: 900,
```

Comment:
900 seconds is 15 minutes, is that the expected value? I thought I heard 5 minutes, but this can be reduced.

Reply:
Hmm, I thought I heard 15 minutes... but IMO we can nail this down as a part of #881

Reply:
Yeah, merge this thing.